docs/user-guide/cli-using-daemon-mode.md

```diff
@@ -1,46 +1,61 @@
 # Using Daemon Mode (Technical Preview)
 
-<Badge text="Technical Preview"/> Daemon mode is available in the `@next` version of Zowe CLI. The Zowe CLI @next release is a technical preview. Technical previews are for testing only and are not ready for production. Your feedback is valued and appreciated.
+<Badge text="Technical Preview"/> Daemon mode is available in the `@next` version of Zowe CLI. The Zowe CLI @next release is a technical preview. Technical previews are for only testing and are not ready for production. Your feedback is valued and appreciated.
 
 If you already installed the supported version `@zowe-v1-lts`, switch versions to try this feature. Daemon mode will be included in the next major Zowe release, V2.0.0-LTS. You can also [install the @next release of Zowe CLI](cli-install-cli-next.md#install-zowe-cli-from-a-download).
 
-**Table of Contents:**
+## Feature overview
 
-- [Feature overview](#feature-overview)
-- [Install the zowe native executable](#install-the-zowe-native-executable)
-- [Running zowe commands with the zowe executable](#running-zowe-commands-with-zowe-executable)
+Daemon mode significantly improves the performance of Zowe CLI commands by running Zowe CLI as a persistent background process (daemon). Running Zowe CLI as daemon lets Zowe absorb the one-time startup of Node.js modules, which resusts in significantly faster responses to Zowe commands.
 
-## Feature overview
+**Important:** We do not recommend using daemon mode in an environment where multiple users use the same system. For example, a shared Linux server.
+
+## Installation notes
 
-Daemon Mode significantly improves the performance of Zowe CLI commands by running Zowe CLI as a persistent background process (daemon).
+The following notes apply to running Zowe in daemon mode:
 
-## Install the zowe native executable
+- To enable daemon mode, ensure that you installed [the @next release of Zowe CLI](cli-install-cli-next.md).
+- When you are running Zowe on a Windows operating system in a virtual environment (for example, Windows Sandbox), you might receive an error message that indicates that a library named VCRUNTIME140.dll is missing. To correct the error, install Visual C++ Redistributable for Visual Studio 2015. For more information, see [Download Visual C++ Redistributable for Visual Studio 2015](https://www.microsoft.com/en-us/download/details.aspx?id=48145).
```

Comment:
Could we add another bullet here, related to an issue that was discovered today when testing daemon mode:

* When you are running Zowe using the Git Bash terminal on a Windows operating system, you may see special characters displayed using the wrong code page. For example, the default codepage 437 renders a form-feed character (`\f`) as an emoji instead (♀️). To fix this, run the command `chcp.com 65001` to change the codepage to UTF-8. If you want this change to be persistent, you can add it to your .bashrc file.